consensus/types/src/attestation.rs

```diff
@@ -350,6 +370,64 @@ impl<E: EthSpec> AttestationElectra<E> {
             Ok(())
         }
     }
+
+    pub fn to_single_attestation_with_attester_index(
+        &self,
+        attester_index: usize,
+    ) -> Result<SingleAttestation, Error> {
+        let Some(committee_index) = self.committee_index() else {
+            return Err(Error::InvalidCommitteeIndex);
+        };
+
+        Ok(SingleAttestation {
+            committee_index: committee_index as usize,
+            attester_index,
+            data: self.data.clone(),
+            signature: self.signature.clone(),
+        })
+    }
+
+    pub fn to_single_attestation(
+        &self,
+        committee: Option<BeaconCommittee>,
```

Comment:
This function appears to be used once and committee is always Some
```suggestion
        committee: BeaconCommittee,
```

Reply:
I've deleted this as its only used in one of the tests